_posts/en/newsletters/2018-11-20-newsletter.md

```diff
@@ -0,0 +1,319 @@
+    her channel with Bob and 7.5% through her channel with Dan (who also
+    has a channel with Charlie).  Although this increases the complexity
+    of implementations somewhat due to having to deal with some partial
+    payments failing some of the time, it doesn't change the security as
+    all payments can use the normal LN security mechanism of releasing a
+    pre-image in order to claim payments (in this case, claiming
+    multiple payments with the same preimage rather than just one).
+
+    For more information, see the following Lightning-Dev threads
+    which often call this feature Atomic
+    Multi-path Payments (AMP): [1][roasbeef amp], [2][zmn base amp],
+    [3][pickhardt local amp].
+
+- **Dual-funded channels:** a nice feature of the current implementation
+  is that only one party to the channel needs to initially commit any
+  funds to it.  For example, Alice opens a channel to Bob with 0.1 BTC
+  of her money and none of Bob's money.  This makes it very easy for
+  users to accept new incoming channels as they don't cost the receiving
+  user anything---not even any opportunity costs (except for operating
+  the LN software).  However, this also means that channels can only be
+  used in one direction initially---Alice can pay Bob or route payments
+  through Bob, but Alice can't receive payments from Bob or from any
+  routing path including Bob until Alice has sent Bob some money.  This
+  creates a bootstrapping problem: if Alice wants to receive payments
+  via LN, she has to get people to open new channels to her node---which
+  requires paying possibly-high onchain fees and waiting for onchain
+  confirmation that can take hours.
+
+    A proposed solution to this problem is to allow dual-funded
+    channels.  Alice agrees to put 0.1 BTC into a channel with Bob if
+    Bob agrees to open the channel with 0.1 BTC of his own funds.  This
+    can cost Bob money---namely onchain transaction fees and lost
+    opportunity cost from having his funds committed for some time---but
+    Bob also receives the opportunity to earn LN routing fees for any
+    payments sent to Alice.
+
+    The basic implementation of dual-funding is probably simple (LN
+    nodes already handle bidirectional payments) but creating an
+    incentive mechanism that can reward capital providers like Bob is
+    still being discussed.  For more information, see the following
+    threads: [1][neigut liquidity], [2][zmn liquidity], [3][zmn dual
+    rbf].  See also [Newsletter #21][].
+
+- **Splicing:** when you currently open an LN channel, you can't
+  increase that channel's maximum balance or withdraw funds from that
+  channel without closing the whole channel and opening another between
+  the same parties.  Closing one channel and opening another requires
+  completely stopping all payments between the two parties until an
+  appropriate number of onchain confirmations have been received for the
+  close-and-reopen transaction.
```

Comment:
Even more importantly splice-outs allow you to perform on-chain payments directly out of an off-chain contract. This means that there is no need to differentiate between off-chain and on-chain funds anymore. This goes a long way towards a much simpler UX where users don't have to differentiate, and in combination with AMP it allows us to hide the channel management completely from the user.